cardverificationsheet/src/main/java/com/stripe/android/cardverificationsheet/cardverifyui/CardVerifyActivity.kt

```diff
@@ -133,6 +133,7 @@ open class CardVerifyActivity : SimpleScanActivity<RequiredCardDetails?>() {
                     )
                 )
             setResult(Activity.RESULT_OK, intent)
+            closeScanner()
```

Comment:
is this intended?

Reply:
yes, this was a bug before where it would never close. It showed up during end-to-end testing that this PR fixes.